storage/remote/queue_manager.go

```diff
@@ -398,6 +443,37 @@ func (t *QueueManager) Stop() {
 	t.metrics.unregister()
 }
 
+// Getting the data from a shard and structuring it as a SegmentRecord from a shard
+func (t *QueueManager) getSegmentRecords() SegmentRecord {
+	RecordSegment := t.shards.qm.watcher.SegmentFile
+	RecordEndpoint := t.shards.qm.url
```

Comment:
The URL will not uniquely identify a QueueManager. Instead, you should use `t.storeClient.Name()`.

Reply:
![image](https://user-images.githubusercontent.com/43453127/89633963-7384e680-d859-11ea-98fc-e9f8a9f45d51.png)
These are the values get for the endpoint when trying this. Is this right?

Reply:
Yep. If you don't specify a name (https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) for a queue in the config file the name is defaulted to the hash of the config for that queue.